### PR TITLE
Add English language (#14)

### PR DESCRIPTION
**Pull request by OctoNezd**
ЛОНДОН ИС ЗЕ КАПИТАЛ ОФ ГРЕТ БРИТЕН

### CODE REVIEW

**Comment by lakekue** [comment]
@OctoNezd Пиздато

**Comment by OctoNezd** [comment]
@lakekue не совсем

**Comment by SkyDiverCool** [comment]
@OctoNezd Доделывай

**Comment by OctoNezd** [comment]
Вроде готово

**Comment by ctl** [comment]
`  "*Hi!* I am bot which makes daily raffle. To know more about, me click on menu", `
`To know more about, me click on menu`
`about, me`

**Comment by ctl** [comment]
`Canceled`
Lol wut? AFAIK , Cancelled

**Comment by OctoNezd** [comment]
http://grammarist.com/spelling/cancel/

**Comment by ctl** [comment]
ayy

**Comment by SkyDiverCool** [comment]
@ctl lmao

**Comment by ctl** [comment]
So cancelled is british spelling , but canceled = US one ?

File: lang/en.json [hunk 1]
```diff
@@ -1,3 +1,119 @@
...
+    "auto_spin_off": "Auto raffle is disabled in this chat/",
+    "auto_spin_set_off": "Now auto raffle is disabled in this chat",
+    "auto_spin_still_off": "Auto raffle is not on in this chat",
+    "stats_me": "Your stats:\n*{0}*: {1} time(s)",
+    "stats_user": "Stats of *{0}*: {1} time(s)",
+    "stats_user_short": "*{0}*: {1} time(s)\n",
+    "stats_all": "Stats of this chat: (page {0} of {1})\n",
+    "admin_allow": "Now this user *have* admin rights on this bot",
+    "admin_still_allow": "This user *already have* admin right on this bot",
+    "admin_deny": "Now this user *doesnt have* admin rights on this bot",
```
Comment (ctl) [inline lang/en.json]: Please, change doesnt to doesn't or does not

**Comment by evgfilim1** [comment]
Starting from commit `1fccc9c`, YAML is used in translations instead of JSON, so you have to rewrite your translation.

**Comment by OctoNezd** [comment]
@evgfilim1 You need to die.

**Comment by evgfilim1** [comment]
Does anyone want to continue @OctoNezd's work?

**Comment by OctoNezd** [comment]
заюзай мой формат, он годный же

**Comment by OctoNezd** [comment]
только там массивов нету

**Comment by SkyDiverCool** [comment]
@evgfilim1 Copy and paste from the old file

**Comment by evgfilim1** [comment]
Closing because of inactivity. Reopen if needed.

**Comment by evgfilim1** [comment]
@OctoNezd, you can use [this site](http://weblate.evgfilim1.me/projects/spin_everyday_bot/indev/) to localize the bot if you want

**Pull request closed without merging**